src/main/java/ru/yandex/practicum/filmorate/dao/film/FilmDAO.java

```diff
@@ -182,6 +182,30 @@ public void removeLike(int targetFilmId, int userId) {
                 targetFilmId, userId);
     }
 
+    @Override
+    public Collection<Film> getFilmRecommendation(int userId) {
+        SqlRowSet filmRows = template.queryForRowSet("select * from \"likes\"");
+        if (!filmRows.next()) {
+            String sqlQuery = "SELECT * FROM \"films\"" +
```

Comment:
Для выдачи всех фильмов можно использовать уже существующий метод getFilms().

Reply:
I had to completely delete it, because base on postman tests we should return empty list when there are no likes, so we shouldn't suggest anything to user

Reply:
Ok